.github/workflows/build_release_assets.yml

```diff
@@ -292,6 +292,25 @@ jobs:
           - clearlinux:latest
 
     steps:
+      - name: Install requirements
+        run: |
+          case "${{ matrix.image }}" in
+            debian:*|ubuntu:*)
+              apt update
+              apt install --no-install-recommends -y git
+              ;;
+            rockylinux*)
+              yum install -y git-core
+              ;;
+            opensuse*)
+              zypper install -y git-core
+              ;;
+            clearlinux*)
+              swupd bundle-add libstdcpp
```

Comment:
Maybe add a comment explaining why we need to install this package, otherwise future-us are going to scratch their heads :).